Examples/SampleCat/SampleCat/Screens/ContentView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+struct ContentView: View {
+    @Environment(HealthViewModel.self) var healthViewModel
+
+    var body: some View {
+        @Bindable var healthViewModel = healthViewModel
+
+        TabView {
+            OfferingsView()
+                .tabItem {
+                    Label("Offerings", systemImage: "dollarsign")
+                }
+
+            ProductsView()
+                .tabItem {
+                    Label("Products", systemImage: "shippingbox.fill")
+                }
+        }
+        .fullScreenCover(item: $healthViewModel.blockingError) { error in
+            FullScreenErrorView(error: error)
+        }
+    }
+}
+
+#Preview {
+    ContentView()
```

Comment:
These previews don't work if there's no API key. I think it would be cool to be able to use emerge in this app as well.